cli/scorecard/violations.go

```diff
@@ -74,7 +76,8 @@ func getDataFromFile(config *ScoringConfig, caiDirName string) ([]*validator.Ass
 	for _, objectName := range destinationObjectNames {
 		reader, err := os.Open(filepath.Join(caiDirName, objectName))
 		if err != nil {
-			return nil, err
+			Log.Debug("getDataFromFile", objectName, "File does not exist")
+			continue
```

Comment:
Making each of the file non-mandatory with debug logs because the new asset type options org policy and access policy may not be widely known and used.

Reply:
addressed in 1e1a678